### DISCUSSION TITLE
Working on the new UI. Dark mode inspired from draculatheme.

### DISCUSSION THREAD

**Discussion by SHR1SHAK** [ChrisTitusTech/winutil#1872] [Ideas]
Hey, Guys I just loved this tool, it helps me save so much time doing the tweaks manually.  Here is the first page that I have completed.  Note that the text is just copy-pasted for visualization purposes.  Something like hover to info is not finished to finish eh overall look and the hover to info is just the same. I made some changes to the text colour. Here is the Figma prototype link if you want to check it yourself.  **_[FIGMA LINK](https://www.figma.com/proto/d6ZxzoJE1LOuERhtWc1tRf/winutil?page-id=0%3A1&type=design&node-id=44-2039&viewport=972%2C185%2C0.51&t=aj9BVbULNNgSWC0f-1&scaling=scale-down)_**
Now I have to just learn how to do this in XAML. If someone is interested they can do it. 

https://github.com/ChrisTitusTech/winutil/assets/55125587/bcd7c7c9-1de8-4d6c-ae81-4ce0674aeaee

Please leave feedback, so that I can improve this design more.

**Comment by DryPringleSoup** [comment]
Very clean but what I will say is that it'd improve the overall look if the contrast between the colors was just a bit higher. Also, I think it'd be good idea to bring back the glow for the selection on top (though a lot more subtle than the glow he have right now) and possibly a slight change to the color of the question marks.

Here's my rendition:
![Screenshot 2024-05-04 122522](https://github.com/ChrisTitusTech/winutil/assets/66896208/e76a1cf3-3caa-4607-a550-ba2d935fa057)

Also here's one the changed question marks but idk how to feel about it:
![2024_05_04_0lf_Kleki](https://github.com/ChrisTitusTech/winutil/assets/66896208/fb0b6bd1-375f-433c-970e-628f8c25707e)

**Reply by SHR1SHAK** [reply]
noted.

**Reply by SHR1SHAK** [reply]
The latest version already discarded the glow as predicted. It too distracting.

**Reply by DryPringleSoup** [reply]
Did they? It still there for me, still as incandescently bright as ever
![image](https://github.com/ChrisTitusTech/winutil/assets/66896208/dc1fad97-98ed-4b86-b6b3-f309caa01a18)

**Reply by SHR1SHAK** [reply]
![Screenshot (1)](https://github.com/ChrisTitusTech/winutil/assets/55125587/ade35012-6126-4e2b-8dc0-50a01e0a5db5)

**Reply by SHR1SHAK** [reply]
mine doesn't show that
